_resources/port1.0/group/fortran-1.0.tcl

```diff
@@ -21,6 +21,15 @@ if {${os.major} < 14} {
 	git.cmd                 ${prefix}/bin/git
 }
 
+# https://github.com/fortran-lang/fpm/issues/1059
+# Drop this upon the next release of FPM:
+compiler.blacklist-append \
+                    macports-gcc-14
+if {${os.platform} ne "darwin" || ${os.major} > 9} {
```

Comment:
the os.major check should go together with the os.platform check for darwin; this isn't correct anymore now (I do understand the change, but you should include os.platform after the OR statement)

Reply:
Someone suggested me this form (maybe @nilason but not sure), and it looked logical: once not-darwin is filtered out, what is left is surely darwin only. Just to understand the issue, why won’t it work?

I will make the change, of course, no argument on that.

Reply:
ah, I did misread that - should be good indeed

Reply:
Ok, then I leave it as is.